package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "movies-search",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.14.1",
+    "@testing-library/react": "^11.2.7",
+    "@testing-library/user-event": "^12.8.3",
```

Comment:
This is not part of this task, but these dependencies should be dev dependencies.